k9s.yaml

```diff
@@ -23,16 +20,18 @@ pipeline:
 
   - uses: go/bump
     with:
+      deps: github.com/mholt/archiver/v3@v3.5.1
       replaces: github.com/mholt/archiver/v3=github.com/anchore/archiver/v3@v3.5.2
 
   - uses: go/build
     with:
       packages: .
-      ldflags: -s -w -X github.com/derailed/k9s/cmd.version=v${{package.version}} -X github.com/derailed/k9s/cmd.commit=$(git rev-parse HEAD) -X github.com/derailed/k9s/cmd.date=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+      ldflags: |
+        -X github.com/derailed/k9s/cmd.version=v${{package.version}}
+        -X github.com/derailed/k9s/cmd.commit=$(git rev-parse HEAD)
+        -X github.com/derailed/k9s/cmd.date=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
       output: k9s
 
-  - uses: strip
-
```

Comment:
Why we are removing these?

Reply:
We can omit this because stripping will be done by the go/build pipeline. ref: https://github.com/chainguard-dev/melange/blob/5ce2bb4ea7969d8a65f43c154a1f597872a7d844/pkg/build/pipelines/go/build.yaml#L58-L62